### PR TITLE
Bookstore: Add styling

### PR DESCRIPTION
**Pull request by ForHemer**
Add styling to the project.  See [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.mdl) for more details.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,12 +1,13 @@
-.cms-header {
+header.cms-header {
   display: flex;
   height: 6rem;
```
Comment (elmar8287) [inline src/App.css:3]: - [ ] Please, consider fixing some points in styling - you are almost done 👍 

1. Please, align the title "Bookstore CMS" and links right, in the same line with the books list, as shown on the screenshot.
2. Please, consider fixing in one line the "Bookstore CMS" and links, as shown on the screenshot.
3. Please, add dividers for each book, as shown on the screenshot.
4. Try to decrease the space between books in the list, to be closer to the original design.
5. Try to move to the left of the account icon, as shown on the screenshot, to be on the same line with the books list.

Screen Shot 2022-04-01 at 11 40 17

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @ForHemer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ] No linter issues 👍 
- [ ] Nice README file 👍 

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (ForHemer) [reply src/App.css:3]: > * [ ]  Please, consider fixing some points in styling - you are almost done +1
> 
> 
>     1. Please, align the title "Bookstore CMS" and links right, in the same line with the books list, as shown on the screenshot.
> 
>     2. Please, consider fixing in one line the "Bookstore CMS" and links, as shown on the screenshot.
> 
>     3. Please, add dividers for each book, as shown on the screenshot.
> 
>     4. Try to decrease the space between books in the list, to be closer to the original design.
> 
>     5. Try to move to the left of the account icon, as shown on the screenshot, to be on the same line with the books list.
> 
> 
> Screen Shot 2022-04-01 at 11 40 17

Hi @elmar8287, Thank you for your suggestion.  Everything is ok now

Screenshot from 2022-04-01 14-09-45
.

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @ForHemer,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ForHemer** [comment]
> # **Approved heavy_check_mark**
> 
> Hi @ForHemer,
> 
> Congratulations on completing this project! 👏🏼 tada
> 
> You have correctly implemented all of the requirementspage_with_curl correctly and without any errorswarning or bugsbeetle.
> 
> Cheers and good luck in your next milestones! four_leaf_clover
> 
> Feel free to leave any questions or comments in the PR thread(don't forget to tag me smile) if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for approving @elmar8287

**Pull request merged**